analytics-recordings.php

```diff
@@ -131,34 +131,43 @@
             }, 0);
         };
 
+        $totaleventswithrecordings = count(array_filter($events, function($event){
```

Comment:
could this be an array_reduce with an increment? might be nicer to read, not sure